views/deductions/trillium-rent-amount.pug

```diff
@@ -11,11 +11,9 @@ block content
   include ../_includes/backButton
 
   div
-    p #{__('To get the most out of your')} <strong>#{__('Ontario Trillium Benefit')}</strong>, #{__('you can enter the amount you paid in rent in the current tax year.')}
+    p #{__('To get the most out of your')} <strong>#{__('Ontario Trillium Benefit')}</strong>, #{__('enter the amount you paid in rent in 2018.')}
 
-    p #{__('Enter the')} <strong>#{__('total annual rent')}</strong>, #{__('you paid for your main residence (including a private long-term care home) in 2018.')}
-
-    p #{__('Leave the field as')} <strong>0</strong>, #{__('if you paid no rent in 2018 or if you don’t have the receipts to verify this information later.')}
+    p #{__('Please note you may be asked to provide receipts to verify this information later.')}
 
```

Comment:
are we planning to add yes/no for property tax as well?

Reply:
yeah totally.